### PR TITLE
REFPLTB-3201 : Create RPI4 Reference Broadband Kirkstone JSON for BundleGen templates/generic

### PR DESCRIPTION
**Pull request by SsandhyaR**
Reason for change : Creating the json files to support RPI4 kirkstone both 32 bit and 64 bit broadband bundles
Test procedure : Created oci bundles and tested with DobbyTool
Risk : Low

### CODE REVIEW

File: templates/generic/rpi4_64_reference_bb_kirkstone.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            "Controller",
+            "DeviceInfo",
+            "Monitor",
+            "OCDM",
+            "TraceControl",
+            "WifiManager",
+            "com.comcast.CoPilot",
+            "com.comcast.DeviceProvisioning",
+            "com.comcast.FrameRate",
+            "com.comcast.HdcpProfile",
```
Comment (cpokuru) [inline templates/generic/rpi4_64_reference_bb_kirkstone.json:20]: remove all video specific items in all files

Comment (SsandhyaR) [reply templates/generic/rpi4_64_reference_bb_kirkstone.json]: Addressed Chandra
Please review

File: templates/generic/rpi4_64_reference_bb_kirkstone.json [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        "variant": "v8"
+    },
+    "rdk": {
+        "version": "2024Q3",
+        "supportedFeatures": [
+            "Controller",
+            "DeviceInfo",
+            "WifiManager",
+            "com.comcast.DeviceProvisioning",
+            "org.rdk.DeviceDiagnostics",
```
Comment (cpokuru) [inline templates/generic/rpi4_64_reference_bb_kirkstone.json:15]: keep things which are applicable for broadband,still few changes are related to video

Comment (SsandhyaR) [reply templates/generic/rpi4_64_reference_bb_kirkstone.json]: Addressed Chandra.
Please review

**Review by cpokuru — COMMENTED** [review]
Required Liberty review

**Review by cpokuru — APPROVED** [review]

**Review by cpokuru — COMMENTED** [review]
Require component owner feedback

**Review by goruklu — APPROVED** [review]

**Pull request merged**